crm_action/models/crm_lead.py

```diff
@@ -26,10 +26,10 @@
 class CrmLead(models.Model):
     _inherit = 'crm.lead'
 
-    def count_actions(self):
+    def _compute_count_actions(self):
         self.actions_count = len(self.action_ids)
 
-    actions_count = fields.Integer(compute='count_actions')
+    actions_count = fields.Integer(compute='_compute_count_actions')
```

Comment:
The standard name, now that you take the time to rename the method, would be `_compute_actions_count`.